data/snehap02.json

```
@@ -0,0 +1,25 @@
+{
+  "name: "Sneha Purkayastha",
```

Missing closing string.
```suggestion
  "name": "Sneha Purkayastha",
```